README.md

```diff
@@ -329,6 +332,45 @@ Take `Rereco17_94X` as an example.
     },
 ```
 
+</p>
+</details>
+
+### Use central maintained jsonpog-integration
+The official correction files collected in [jsonpog-integration](https://gitlab.cern.ch/cms-nanoAOD/jsonpog-integration) is updated by POG except `lumiMask` and `JME` still updated by maintainer. No longer to request input files in the `correction_config`.  
```

Comment:
JME corrections are also part of jsonpog, would be nice to have them as well (not necessarily in this PR, but in a follow-up one)

Reply:
indeed for JME one it's not possible with CorrectMETFactory and CorrectJetFactory. we need to follow up the development from coffea group

Reply:
Is there an ongoing discussion or issue about this in the coffea group?

Reply:
Not really in the issues or discussion on git, but I would probably bring this up

Reply:
Would be great if you could do that! Once done, you can link the URL here